plugins/language_autohotkey_v1.lua

```diff
@@ -0,0 +1,155 @@
+-- mod-version:2 -- lite-xl 2.0
```

Comment:
Could you update this with the new mod-version?
```suggestion
-- mod-version:3
```